ts/types/IgeEntity.d.ts

```diff
@@ -1,11 +1,14 @@
 declare class IgeEntity extends IgeObject {
 
+	_alive: boolean;
 	_destroyed: boolean;
 	_deathTime: number;
 	_category: string; // TODO more specific values
 	_translate: IgePoint3d;
 	_rotate: IgePoint3d;
+	_scale: IgePoint3d;
 	_stats: EntityStats;
+	_bounds2d: IgePoint3d;
```

Comment:
correct type is `IgePoint2d`

Reply:
there are some really unfortunate places in the source that I have found in the past where IgePoint3d is used and the rotation value is passed as the z coord. If we can remove these as we find them that's awesome, but we'll have to make note of all these.